build_tools/python/e2e_test_framework/models/torch_models.py

```diff
@@ -52,3 +52,94 @@
     "https://storage.googleapis.com/iree-model-artifacts/pytorch/torch_models_20230307.103_1678163233/SD_UNET_MODEL/linalg.mlir",
     entry_function="forward",
     input_types=["1x4x64x64xf32", "1x77x768xf32"])
+
+# Converted from https://pytorch.org/vision/main/models/generated/torchvision.models.resnet50.html
+RESNET50_1x3x224x224_FP32_TORCH = common_definitions.Model(
+    id=unique_ids.MODEL_RESNET50_1x3x224x224_FP32_TORCH,
+    name="Resnet50PTBatch1",
+    tags=["fp32", "cnn", "batch-1"],
+    source_type=common_definitions.ModelSourceType.EXPORTED_LINALG_MLIR,
+    source_url=
+    "https://storage.googleapis.com/iree-model-artifacts/pytorch/torch_models_20230321.784_1679461251/RESNET50/batch_1/linalg.mlir",
+    entry_function="forward",
+    input_types=["1x3x224x224xf32"])
+
+RESNET50_8x3x224x224_FP32_TORCH = common_definitions.Model(
+    id=unique_ids.MODEL_RESNET50_8x3x224x224_FP32_TORCH,
+    name="Resnet50PTBatch8",
+    tags=["fp32", "cnn", "batch-8"],
+    source_type=common_definitions.ModelSourceType.EXPORTED_LINALG_MLIR,
+    source_url=
+    "https://storage.googleapis.com/iree-model-artifacts/pytorch/torch_models_20230321.784_1679461251/RESNET50/batch_8/linalg.mlir",
+    entry_function="forward",
+    input_types=["8x3x224x224xf32"])
+
+RESNET50_16x3x224x224_FP32_TORCH = common_definitions.Model(
```

Comment:
nit: "x" in the middle should be capitalized?

Reply:
Changed to X